### ISSUE TITLE
Monthly Milestone - November 2018 🍁 🌾

### ISSUE THREAD

**Issue by dhmlau** [strongloop/loopback-next#1961]
## November Milestone
- Focus on user adoption: focus on questions/bugs/functional parity requests from users
- Improve documentation and improve docs related to contribution to LB

### LB 4.0
- [x] API Explorer epic  https://github.com/strongloop/loopback-next/issues/559
  - [x] Support local API Explorer  https://github.com/strongloop/loopback-next/issues/1960
- [ ] Docs: make it easier to contribute
  - [ ] How to contribute documentation: https://github.com/strongloop/loopback-next/issues/1853
  - [x] Update the contribution page for Loopback 3: https://github.com/strongloop/loopback-next/issues/1673
  - [ ] [Docs] Update ways to contribute to LoopBack: https://github.com/strongloop/loopback-next/issues/1228
- [ ] Docs clean up https://github.com/strongloop/loopback-next/issues/1951
  - [ ] 🚶(Spike) LB4 Documents need a cleanup pass, https://github.com/strongloop/loopback-next/issues/1908
  - [x] Update LTS status of old loopback packages, https://github.com/strongloop/loopback-next/issues/1802
  - [x] 🚶Review todo list tutorial  https://github.com/strongloop/loopback-next/issues/1828
- [ ] Bugs/Requests from community
  - [ ] 🚶[Spike] How to do relation traversal #1952
  - [ ] 🚶hasOne https://github.com/strongloop/loopback-next/issues/1422 (blocked by findOrCreate task)
  - [ ] ~findOrCreate https://github.com/strongloop/loopback-next/issues/1956~
  - [x] Add the doc for datasource migration, https://github.com/strongloop/loopback-next/issues/1547 

### LB 3
- [ ] 🚶Fix the skipped test cases after upgrading to yeoman-generator@3.x https://github.com/strongloop/generator-loopback/issues/389
- [x] Publish new release of loopback-cli https://github.com/strongloop/loopback-cli/issues/76
- [x] Get mongodb connector to new semver-major version https://github.com/strongloop/loopback-connector-mongodb/issues/476 

### Stretch goals
- [ ] Update LB workspace to LB3 https://github.com/strongloop/loopback-workspace/issues/498
- [x] Remove LB2 in APIConnect v5 https://github.com/strongloop-internal/scrum-apex/issues/398
- [ ] How to report bugs and create reproducible example app: https://github.com/strongloop/loopback-next/issues/1854
- [ ] [Docs] Relation decorators page should be updated  https://github.com/strongloop/loopback-next/issues/1910
- [x] Change banner for loopback.io documentation site and apidocs.loopback.io https://github.com/strongloop/loopback.io/issues/757
- [ ] Clean up "Contributing to LoopBack" page https://github.com/strongloop/loopback.io/issues/758
x

**Comment by dhmlau** [comment]
Has moved the slipped items to [Dec milestone](https://github.com/strongloop/loopback-next/issues/2084).  Closing this milestone ticket.
